docs/api-docs/cart-and-checkout/open-source-checkout.md

```diff
@@ -67,6 +67,19 @@ You can upload a custom checkout to your store's server using WebDAV. The instru
 
 ## Installing custom checkouts  
 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--note">
+<div class="HubBlock-content">
+  
+  <!-- theme: note -->
+> 1. Please get in touch with support if you wish to customize checkout. 
```

Comment:
It feels like this is missing the context of why you need to reach out to Support, which is that there is "a bug where the CSS of custom checkout gets overridden by the CSS of default checkout." Maybe the wording changes, but developers need to know why they are reaching out so that they know what to request.